composer.json

```diff
@@ -22,21 +21,18 @@
     "require": {
         "php": "^5.6 || ^7.0",
         "guzzlehttp/psr7": "^1.4",
-        "guzzlehttp/guzzle": "^6.0",
-        "symfony/options-resolver": "~2.6 || ~3.0 || ^4.0",
         "altapay/api-php":"^3.0",
         "psr/log": "^1.1",
-        "symfony/dependency-injection": "^5.2",
-        "psr/event-dispatcher": "^1.0",
-        "laminas/laminas-httphandlerrunner": "^1.3",
-        "szepeviktor/phpstan-wordpress": "^0.7.1",
+        "szepeviktor/phpstan-wordpress": "^0.7.1"
```

Comment:
Move this to require-dev as it is a developer tool and not part of what we want users to install